helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -130,13 +130,21 @@ receivers:
 
   {{- if and (eq .Values.logsEngine "otel") .Values.logsCollection.containers.enabled }}
   filelog:
+    {{- if .Values.isWindows }}
+    include: ["C:/var/log/pods/*/*/*.log"]
```

Comment:
Should we have `\\` separators consistently for windows or they don't work in all the cases?

Reply:
Yes we can use `\\` in `include` section but in `exclude` we need to use `\` to so for consistency I have used `/`.
if you want then I can change separators to `\\` in include and in `exclude` we have to make following change
from
```
exclude: 
  - C:/var/log/pods/{{ .Release.Namespace }}_{{ include "splunk-otel-collector.fullname" . }}*_*/otel-collector/*.log
```
to
```
exclude: [ "C:\\var\\log\\pods\\{{ .Release.Namespace }}_{{ include "splunk-otel-collector.fullname" . }}*_*\\otel-collector\\*.log" ]
```
Should I?

Reply:
I don't get why exclude section is different? Probably only quotes are needed but list representation can stay the same like
```
- "C:\\var\\log\\pods\\{{ .Release.Namespace }}_{{ include "splunk-otel-collector.fullname" . }}*_*\\otel-collector\\*.log" ]
```

Reply:
Yes, you are right only quotes are needed in exclude section. 
Now I made a change to make separators consistent with `\\`